ddtrace/opentracer/span_context.py

```diff
@@ -1,5 +1,10 @@
 from opentracing import SpanContext as OpenTracingSpanContext
 
+from ddtrace.context import Context
```

Comment:
I guess this is a conflict not properly solved? I see 3 times the same import.

Reply:
Must be. Haha.